am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -51,18 +52,36 @@ public static ExplicitAccessGrant createGrant(String resourceId,
         return createGrant(resourceId, ImmutableSet.of(accessorId), attributePermissions);
     }
 
+    public static ExplicitAccessGrant createGrant(String resourceId,
```

Comment:
Delegate grant creation to this method in other overloaded methods. No point having two places where whole object is built.

Reply:
👍